README.md

```diff
@@ -12,7 +12,50 @@ If you are new to the Windows Package Manager, you might want to [Explore the Wi
 
 ### Microsoft Store [Recommended]
 
-The client is distributed within the [App Installer](https://www.microsoft.com/p/app-installer/9nblggh4nns1) package. 
+The client is distributed within the [App Installer](https://www.microsoft.com/p/app-installer/9nblggh4nns1) package.
+
+### Microsoft Store using PowerShell
+
+```PowerShell
+$applicationId = "9NBLGGH4NNS1" # Microsoft.DesktopAppInstaller_8wekyb3d8bbwe
+$skuId = 0016
+
+# Obtain packageFamilyName from applicationId
```

Comment:
Why do we need to obtain the packageFamilyName if it is already known?

Reply:
> I think this would probably be better as a separate file, so that it can be downloaded and run directly or called using `Invoke-WebRequest`. Including large blocks of script in README is often a deterrent for non-technical people to get the information they are looking for.

I agree - let me prepare a separate installation script. I suggest using simple `install.ps1` - is that ok?
 
> Another point to consider is that this relies on an API, which could be deprecated or updated at any point in time. Since documentation in the readme is often one of the lower priority items, it could be a while before anyone notices the install script is broken; If the script is in a separate file then it would be easier for users to create an issue against, since it would be clearer where the exact issue is.

Good point! Installation script in the dedicated file will have better maintainability.